IBM.Watson.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2036
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.31112.23
 MinimumVisualStudioVersion = 10.0.40219.1

```

Comment:
do we need this file?

Reply:
Yes, its the main solution for all of the projects. I updated my visual studio so the version updated. I dont think its a big deal.